lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -245,18 +245,18 @@ defmodule LightningWeb.WorkflowLive.Edit do
                       </div>
                     <% else %>
                       <div class="flex flex-col items-center justify-center h-full">
-                        <div class="text-center">
-                          <p class="text-gray-500 text-sm">
+                        <div class="text-center w-1/2">
```

Comment:
@josephjclark Contacting admins is relevant information for users in organizations where they might need to use the AI feature and it's not configured. it is important for them to know there is someone who can help and we have not locked it as a paid feature. 

Please can we revert the language?